DICOMwebBrowser/DICOMwebBrowser.py

```diff
@@ -1090,12 +1091,25 @@ def onCancel(self):
 
 class GoogleCloudPlatform(object):
 
+  def findgcloud(self):
+    gcloudPath = shutil.which('gcloud')
+    if gcloudPath:
+      return gcloudPath
+
+    if sys.platform in ("linux", "darwin"):
+      # Try a second way to find gcloud path
+        shell = os.environ.get("SHELL")
+        if shell:
+          process = slicer.util.launchConsoleProcess([shell, "-c", "which gcloud"])
+          process.wait()
+          gcloudPath = process.stdout.read().strip()
+          if os.path.exists(gcloudPath):
+            return gcloudPath
+
+    logging.error(f"Unable to locate gcloud, please install the Google Cloud SDK")
```

Comment:
This matches the previous behavior, but should we raise an exception here instead of returning None and later running a process with args [None, ...]

Reply:
@vanossj did this work for you without the `-i` flag?  In my experiments I `-i` that because path is set in .zshrc.  I think bash works the same and doesn't read .bashrc unless it's an interactive shell.

Reply:
it did work for me without the `-i` flag, but that didn't really make sense. You should need an interactive shell to source the .bashrc/.zshrc scripts into the environment.

I added the `-i` flag, but that causes the terminals to spew out lots of messages and warnings about not actually being connected to a terminal, or letting you know there are oh-my-zsh updates to get.

I just search through each line until i find one that is a valid path to `gcloud`. For my testing, its always been the second to last line, the last being and empty string. But searching through a couple of strings is probably the more robust method than assuming the second to last line is the output of `which gcloud` for all shells and shell configurations.